testing/libfixposix/APKBUILD

```diff
@@ -2,7 +2,7 @@
 # Contributor: Will Sinatra <wpsinatra@gmail.com>
 pkgname=libfixposix
 pkgver=0.4.3
-pkgrel=0
+pkgrel=1.0
```

Comment:
Why is the pkgrel `1.0` instead of `1`? I don't think this is valid.

Reply:
I honestly couldn't remember if it was 1.0 or 1, but that's an easy fix at least.

Reply:
Regardless, it's set to 1 instead of 1.0 now.